vm/devices/vmbus/vmbus_server/src/channels.rs

```diff
@@ -2237,18 +2247,20 @@ impl<'a, N: 'a + Notifier> ServerWithNotifier<'a, N> {
             .find(|v| request.version_requested == **v as u32)
             .copied()?;
 
+        // The max version may be limited in order to test older protocol versions.
```

Comment:
Worth separating out and backporting to release/2411? Or does this not matter in practice?